sos/cleaner/__init__.py

```diff
@@ -696,6 +730,19 @@ def obfuscate_report(self, archive):  # pylint: disable=too-many-branches
                     # is used. In this case we want to make sure
                     # the cleaner doesn't try to clean a binary file
                     continue
+                is_certificate = file_is_certificate(fname)
+                if is_certificate:
+                    if is_certificate == "certificatekey":
```

Comment:
In theory, key files should have been removed by report already. But surely they can remain also here, makes sense to remove them despite they dont usually contain any *cleaner*-sensitive data.